tests/daily_scraper/test_daily_scraper.py

```diff
@@ -19,6 +19,9 @@
     ({'platform': 'win64'},
      '2013-10-01-03-02-04-mozilla-central-firefox-27.0a1.en-US.win64.installer.exe',
      'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-central/firefox-27.0a1.en-US.win64.installer.exe'),
+    ({'platform': 'win-arm64'},
+     '2013-10-01-03-02-04-mozilla-central-firefox-27.0a1.en-US.win64-aarch64.installer.exe',
+     'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-central/firefox-27.0a1.en-US.win64-aarch64.installer.exe'),
```

Comment:
@Cauacs given that those aarch64 builds for Windows are also available for beta/release builds, would you mind also adding tests for the candidate and release scrapers? That would give us full coverage. 

If you don't have the time to do that please let me know and I'll get them added. Thanks!